vale/styles/jargon/Superlative.yml

```diff
@@ -0,0 +1,398 @@
+---
```

Comment:
The two superlative files could/should be moved out of jargon package

More information about this in #11 

- #11 

Thanks @alexvonme for bringing this to my attention

Reply:
I killed those files